pennylane_ls/SingleQuditDevice.py

```diff
@@ -32,15 +33,16 @@ class SingleQuditDevice(Device):
 
     _observable_map = {"Lz": Lz, "Z": Z, "Lz2": Lz2}
 
-    def __init__(self, shots=1, username=None, password=None):
+    def __init__(self, shots=1, username=None, password=None, job_id=None, blocking=False):
         """
         The initial part.
         """
         super().__init__(wires=1, shots=shots)
         self.username = username
         self.password = password
-        self.url_prefix = "https://qsimsim.synqs.org/shots/"
-
+        self.url_prefix = "http://qsim-atomic-drop.herokuapp.com/singlequdit/"#"http://qsimsim.synqs.org/shots/"#
```

Comment:
In one of the other devices we are actually able to set the URL if it is given. This would seem like a cleaner solution no ?